Frontend/src/components/Home.tsx

```diff
@@ -10,58 +10,115 @@ interface Props {
 
 const Home = ({ navigation }: Props) => {
   const { contextState } = useContextState();
-
-  const handleContinue = () => {
-    navigation.navigate(Routes.Chat);
+  const [showSignup, setShowSignup] = useState(true);
+  const [showUnlock, setShowUnlock] = useState(false);
+  
+  
+  const handleLoginFinger = () => {
+    navigation.navigate(Routes.LoginFingerprint);
   };
-
-  const handleLogin = () => {
+  const handleLoginPassword = () => {
     navigation.navigate(Routes.Login);
   };
 
+  const handleLoginFace = () => {
+    
```

Comment:
Cuando se deja algo sin terminar porque se tiene que hacer en otra tarea está bueno dejar un comentario TODO. Por ej dejaría:
`// TODO: navigate to face login page`